src/waveform/renderers/waveformsignalcolors.cpp

```diff
@@ -55,6 +55,12 @@ bool WaveformSignalColors::setup(const QDomNode &node, const SkinContext& contex
         m_playPosColor = m_axesColor;
     }
 
+    m_playedOverlayColor = rgbaColorFromString(context.selectString(node, "PlayedOverlayColor"));
+    if (!m_playedOverlayColor.isValid()) {
+        m_playedOverlayColor = QColor(0,0,0,200);
+    }
+    m_playedOverlayColor = WSkinColor::getCorrectColor(m_playedOverlayColor).toRgb();
+
     m_bgColor.setNamedColor(context.selectString(node, "BgColor"));
```

Comment:
While you're at it, could you use the new ``rgbaColorFromString`` function here?